docs/user/dashboard/dashboard.asciidoc

```diff
@@ -0,0 +1,472 @@
+
+
+a| *Tree map*
+
+Relates different segments of your data to the whole. Each rectangle is subdivided into smaller rectangles, or sub branches, based on 
+its proportion to the whole. Use treemaps to make efficient use of space to show percent total for each category.
+
+| image:images/treemap.png[Tree map]
+
+
+a| *Heat map*
+
+Displays graphical representations of data where the individual values are represented by colors. Use heat maps when your data set includes 
+categorical data. For example, use a heat map to see the flights of origin countries compared to destination countries using the sample flight data.
+
+| image:images/heat_map.png[Heat map]
+
+a| *Goal*
+
+Displays how your metric progresses toward a fixed goal. Use the goal to display an easy to read visual of the status of your goal progression.
+
+| image:images/goal.png[Goal]
+
+
+a| *Gauge*
+
+Displays your data along a scale that changes color according to where your data falls on the expected scale. Use the gauge to show how metric 
+values relate to reference threshold values, or determine how a specified field is performing versus how it is expected to perform.
+
+| image:images/gauge.png[Gauge]
+
+
+a| *Metric*
+
+Displays a single numeric value for an aggregation. Use the metric visualization when you have a numeric value that is powerful enough to tell 
+a story about your data.
+
+| image:images/metric.png[Metric]
+
+
+a| *Data table*
+
+Displays your raw data or aggregation results in a tabular format. Use data tables to display server configuration details, track counts, min, 
+or max values for a specific field, and monitor the status of key services.
+
+| image:images/data_table.png[Data table]
+
+
+a| *Tag cloud*
+
+Graphical representations of how frequently a word appears in the source text. Use tag clouds to easily produce a summary of large documents and 
+create visual art for a specific topic.
+
+| image:images/tag_cloud.png[Tag cloud]
+
+
+a| *Maps*
+
+For all your mapping needs, use <<maps,Maps>>.
+
+| image:images/maps.png[Maps]
+
+
+|===
+
+[float]
+[[create-panels]]
+== Create panels
+
+To create a panel, make sure you have {ref}/getting-started-index.html[data indexed into {es}] and an <<index-patterns,index pattern>> 
+to retrieve the data from {es}. If you aren’t ready to use your own data, {kib} comes with several pre-built dashboards that you can test out. For more information, 
+refer to <<add-sample-data, Explore {kib} using sample data>>.
+
+To begin, click *Create new*, then choose one of the following options on the 
+*New Visualization* window:
+
+* Click on the type of panel you want to create, then configure the options. 
+
+* Select an editor to help you create the panel. 
+
+[role="screenshot"]
+image:images/Dashboard_add_new_visualization.png[Example add new visualization to dashboard]
+
+{kib} provides you with several editors that help you create panels.
+
+[float]
+[[lens]]
+=== Create visualization panels with Lens
```

Comment:
This is confusing because above, you've listed "types of panels" but it doesn't say that "visualization" is a type of panel.